atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -131,6 +156,7 @@ def run_in_orchestrator(self):
         pullspecs = self._get_pullspecs(operator_manifest.csv, should_skip)
 
         if operator_manifest.csv.has_related_images() or should_skip:
+            # TODO do we want to support CSV modifications also in this case?
```

Comment:
:eyes:

Reply:
well freshmaker might rebuild any bundle I'd say, so even when related images where already specified, freshmaker wants rebuild with new pullspecs, WDYT @lcarva

Reply:
I'd address this in separate PR to unblock rest of the work

Reply:
Freshmaker will not rebuild bundle images where the relatedImages section was not filled in by OSBS. It currently checks for the value of `True` for the `created_by_osbs` key.

Reply:
Thanks! Added explicit check that this is not allowed (yet?)

Reply:
well so if it isn't supported we should fail here in that case

Reply:
It works better when the one push local changes :D